EDEngineer/Resources/Data/localization.json

```diff
@@ -2090,6 +2090,24 @@
       "de": "@Synthese",
       "fr": "@Synthèse",
       "ru": "@Синтез"
+    },
+    "Launch Local API": {

```

Comment:
new localization for upcoming update :)

@VChet 
@calvin00 
@Buchodt 
@TCAS 
@Scorprox 
@Giblood 
@rogerkilo